queue_services/business-emailer/src/business_emailer/email_templates/common/reg-business-info.html

```diff
@@ -5,15 +5,21 @@
   <div>
     <div class="name bold">Business name:</div>
     {% if  filing_type == 'registration' %}
-      {% if filing.nameRequest.legalName %}
+      {% if business_name %}
+      <div clases="value">{{ business_name }}</div>
```

Comment:
`business_name` is not set in `registration` https://github.com/bcgov/lear/blob/main/queue_services/business-emailer/src/business_emailer/email_processors/registration_notification.py